truss/tests/test_model_schema.py

```diff
@@ -113,7 +113,11 @@ def test_truss_with_annotated_inputs_outputs():
 
         assert response.status_code == 400
         assert "error" in response.json()
-        assert "Request Validation Error" in response.json()["error"]
+
+        assert (
+            "Request Validation Error: 1 validation error for ModelInput"
```

Comment:
Is there (and should we assert that) more detailed information, e.g. if it's a type error, missing field, unknown field etc.? I think just saying "there is *some* error" is not helpful or actionable.

Reply:
yeah, I can expand this. fwiw `ModelInput` is the name of the pydantic model